templates/zerver/help/about-streams-and-topics.md

```diff
@@ -12,7 +12,7 @@ message is about.
 Messages with the same stream and topic are shown together as a
 conversational thread. Here is what it looks like in Zulip.
 
-![](/static/images/help/streams-and-topics.png)
+![Streams and Topics](/static/images/help/streams-and-topics.png)
```

Comment:
Our capitalization style is "Sentence case", so this should be "Streams and topics".  The same comment applies to most of the rest of this file.  (Obviously proper nouns should stay capital, so perhaps the above should be "Hello World integration")?

Reply:
Sure, I'll make the changes